compose/docker-compose.acceptance-tests.yml

```diff
@@ -0,0 +1,54 @@
+---
+version: "2"
+
+# Based on "Getting Started with Docker Compose" (https://goo.gl/pBpyyY)
+# More notes here: https://goo.gl/yDLK7f (incl. debugging instructions).
+
+services:
+
+  selenium-hub:
+    image: "selenium/hub:3.7.1-beryllium"
+    expose:
+      - "4444"
+    ports:
+      - "62100:4444"
+    environment:
+      GRID_BROWSER_TIMEOUT: "10"
+      GRID_DEBUG: "false"
+
+  selenium-firefox:
+    image: "selenium/node-firefox-debug:3.7.1-beryllium"
+    links:
+      - "selenium-hub"
+      - "nginx"
+    shm_size: 2g  # See https://goo.gl/dkZzHh
+    ports:
+      - "62102:5900"
+    environment:
+      FIREFOX_VERSION: "47.0.1"
```

Comment:
Is this here for a reason? I noticed that https://hub.docker.com/r/selenium/node-firefox/~/dockerfile/ takes a FIREFOX_VERSION but it's a build argument not an environment variable. Is this being used somewhere else?

Reply:
Ha. I don't even know if this does anything. All I know is that I got the test to pass when this was set. I'll rerun them without this and see if they pass.